tasks/update.py

```diff
@@ -249,10 +239,155 @@ def update(self):
+            if self.setting_class.NEW_LINEITEM_TYPE in ('NETWORK','HOUSE'):
+                if 'primaryGoal' in line_item:
+                    line_item['primaryGoal']['goalType'] = 'DAILY'
+                    line_item['primaryGoal']['units'] = 100
+                if self.setting_class.NEW_LINEITEM_TYPE == "HOUSE":
+                    line_item['priority'] = 16
+
+            elif self.setting_class.NEW_LINEITEM_TYPE in ('SPONSORSHIP'):
+                if 'primaryGoal' in line_item:
+                    line_item['primaryGoal']['unitType'] = 'IMPRESSIONS'
+                    line_item['primaryGoal']['goalType'] = 'DAILY'
+                    line_item['primaryGoal']['units'] = 100
+                line_item['skipInventoryCheck'] = True
+                line_item['allowOverbook'] = True
+                line_item['priority'] = 4
+
+            elif self.setting_class.NEW_LINEITEM_TYPE == "PRICE_PRIORITY":
+                line_item['priority'] = 12
+
+            line_items_with_current_type[line_item['name']]=current_type
+            updated_line_items.append(line_item)
+
+        return updated_line_items, line_items_with_current_type
+
+    def update(self):
+        """
+        Function to update "Line Item Type" of line items in GAM.
+        It retrieves line items from the response, selects line items to update, and then updates them using the LineItemService.
+        The user is prompted for confirmation before proceeding with the update.
+        If the update is successful, the names of the updated line items are logged.
+        """
+        response = self.get_line_items()
+        line_items = response['results'] if 'results' in response else []
+        if len(line_items ) <= 0:
+            self.logger.info("No line item found for given input")
+            return
+
+        line_items_to_update, line_items_with_current_type = self.select_line_items_to_update(line_items)
+        self.print_line_items_with_current_type(line_items_with_current_type)
+
+        if len(line_items_to_update) > 0:
+            user_confirmation = input("Do you want to proceed with the update? (y/n): ").lower()
+            if user_confirmation != 'y':
+                self.logger.info("Update canceled.")
+                return
+
+            line_item_service = self.ad_manager_client.GetService('LineItemService', version=self.API_VERSION)
+            returned_line_items=line_item_service.updateLineItems(line_items_to_update)
+            if len(returned_line_items) > 0:
+                self.logger.info("\nSuccessfully updated following line items")
+                for line_item in returned_line_items:
+                    self.logger.info(f"{self.color.BLUE}{line_item['name']}{self.color.END}")
+            else:
+                self.logger.info("No line items were updated.")
+        else:
+            self.logger.info("No line items matched the criteria for update.")
```

Comment:
Can we make this a separate function?  `VideoPositionUpdater` also have similar update code.